packages/core/core/src/migrations/index.ts

```diff
@@ -0,0 +1,14 @@
+import draftPublishMigrations, { type Input } from './draft-publish';
+import i18nMigrations from './i18n';
+
+const enable = async ({ oldContentTypes, contentTypes }: Input) => {
+  await i18nMigrations.enable({ oldContentTypes, contentTypes });
+  await draftPublishMigrations.enable({ oldContentTypes, contentTypes });
+};
+
+const disable = async ({ oldContentTypes, contentTypes }: Input) => {
+  await i18nMigrations.disable({ oldContentTypes, contentTypes });
```

Comment:
should this be inverted ? 

usually you do a-b<->b-a  ?

Reply:
When disabling is not relevant I think (It's working in this way). discardDraft is going to use the locale defined on the entry because Content Type has not i18n (In this case is going to be always null because i18n migration run first, but otherwise is going to use the old locale)

On enable it should be i18n migration first, because discardDraft adds the default locale because content type has i18n enabled.